airbyte-webapp/src/pages/ConnectorBuilderPage/ConnectorBuilderPage.tsx

```diff
@@ -1,36 +1,64 @@
+import classnames from "classnames";
+import { Formik } from "formik";
 import { useIntl } from "react-intl";
+import { useToggle } from "react-use";
 
+import { Builder } from "components/connectorBuilder/Builder/Builder";
+import { BuilderFormValues } from "components/connectorBuilder/Builder/types";
 import { StreamTestingPanel } from "components/connectorBuilder/StreamTestingPanel";
 import { YamlEditor } from "components/connectorBuilder/YamlEditor";
 import { ResizablePanels } from "components/ui/ResizablePanels";
 
-import { ConnectorBuilderStateProvider } from "services/connectorBuilder/ConnectorBuilderStateService";
+import {
+  ConnectorBuilderStateProvider,
+  useConnectorBuilderState,
+} from "services/connectorBuilder/ConnectorBuilderStateService";
 
 import styles from "./ConnectorBuilderPage.module.scss";
 
 const ConnectorBuilderPageInner: React.FC = () => {
   const { formatMessage } = useIntl();
+  const [showYamlEditor, toggleYamlEditor] = useToggle(false);
+
+  const { builderFormValues } = useConnectorBuilderState();
 
   return (
-    <ResizablePanels
-      className={styles.container}
-      firstPanel={{
-        children: <YamlEditor />,
-        className: styles.leftPanel,
-        minWidth: 100,
-      }}
-      secondPanel={{
-        children: <StreamTestingPanel />,
-        className: styles.rightPanel,
-        flex: 0.33,
-        minWidth: 60,
-        overlay: {
-          displayThreshold: 325,
-          header: formatMessage({ id: "connectorBuilder.testConnector" }),
-          rotation: "counter-clockwise",
-        },
+    <Formik
+      initialValues={builderFormValues}
+      onSubmit={(values: BuilderFormValues) => {
+        console.log(values);
```

Comment:
It is not. This is actually a slightly weird form because we are never really "submitting" it, so I've just changed this to be a no op (since Formik still requires you to specify an onSubmit function)